pkg/web/static/js/customerAccounts.js

```diff
@@ -42,9 +42,13 @@ document.body.addEventListener("htmx:configRequest", (e) => {
   }
 });
 
+// Count number of wallet addresses to create a unique ID as crypto wallets are added and deleted.
+// Start counter at 1 to avoid conflict with the first crypto wallet.
+let walletIdCounter = 1;
```

Comment:
If we need unique IDs, can we use an ID generation library here? Such as https://www.npmjs.com/package/uuid. That would avoid having to rely on the global variable.

Reply:
Alternatively you might be able to use https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID and avoid the dependency.

Reply:
I think that's a much better idea. I'll make the update.